CRITICAL - 2022-01-20 07:35:57 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:36:10 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:36:12 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:36:13 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:36:14 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:36:15 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 07:39:29 --> Invalid file: App\Views\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\auth-...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\auth-...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(429): view('App\\Views\\auth-...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('App\\Views\\auth-...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 08:01:53 --> Trying to get property 'id' of non-object
#0 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Models/UserModel.php(87): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', '/home/u1553325/...', 87, Array)
#1 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(189): Myth\Auth\Models\UserModel->withGroup('mahasiswa')
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->attemptRegister()
#3 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-20 08:05:12 --> Trying to get property 'id' of non-object
#0 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Models/UserModel.php(87): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', '/home/u1553325/...', 87, Array)
#1 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(189): Myth\Auth\Models\UserModel->withGroup('mahasiswa')
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->attemptRegister()
#3 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-20 08:05:19 --> Trying to get property 'id' of non-object
#0 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Models/UserModel.php(87): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', '/home/u1553325/...', 87, Array)
#1 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(189): Myth\Auth\Models\UserModel->withGroup('mahasiswa')
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->attemptRegister()
#3 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2022-01-20 08:07:11 --> Table 'u1553325_pmbci.profil' doesn't exist
CRITICAL - 2022-01-20 08:07:11 --> Table 'u1553325_pmbci.profil' doesn't exist
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Model.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/BaseModel.php(523): CodeIgniter\Model->doFind(true, '3')
#6 /home/u1553325/public_html/pmb/app/Controllers/Home.php(15): CodeIgniter\BaseModel->find('3')
#7 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): App\Controllers\Home->mahasiswa()
#8 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-01-20 08:10:36 --> Class 'App\Models\ProfilModel' not found
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(802): App\Controllers\Pendaftar->__construct()
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(403): CodeIgniter\CodeIgniter->createController()
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-20 08:10:52 --> Class 'App\Models\ProfilModel' not found
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(802): App\Controllers\Pendaftar->__construct()
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(403): CodeIgniter\CodeIgniter->createController()
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-20 08:10:53 --> Class 'App\Models\ProfilModel' not found
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(802): App\Controllers\Pendaftar->__construct()
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(403): CodeIgniter\CodeIgniter->createController()
#2 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-20 10:28:17 --> Invalid file: App\Views\panel\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\panel...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\panel...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(414): view('App\\Views\\panel...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(58): Myth\Auth\Controllers\AuthController->_render('App\\Views\\panel...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-20 10:32:44 --> Invalid file: App\Views\panel\auth-login.php
#0 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('App\\Views\\panel...')
#1 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/Common.php(1089): CodeIgniter\View\View->render('App\\Views\\panel...', Array, true)
#2 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(414): view('App\\Views\\panel...', Array)
#3 /home/u1553325/public_html/pmb/vendor/myth/auth/src/Controllers/AuthController.php(58): Myth\Auth\Controllers\AuthController->_render('App\\Views\\panel...', Array)
#4 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(825): Myth\Auth\Controllers\AuthController->login()
#5 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 /home/u1553325/public_html/pmb/vendor/codeigniter4/framework/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /home/u1553325/public_html/pmb/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
